fcitx-qt5 fails to build on unstable

**Describe the bug**
fcitx-qt5 fails to build on unstable:


```
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/include/FcitxQt5/FcitxQtWidgetsAddons/fcitxqtconfiguiplugin.h
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/include/FcitxQt5/FcitxQtWidgetsAddons/fcitxqtconfiguiwidget.h
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/include/FcitxQt5/FcitxQtWidgetsAddons/fcitxqtkeysequencewidget.h
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/include/FcitxQt5/FcitxQtWidgetsAddons/fcitxqtwidgetsaddons_export.h
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/lib/fcitx/libexec/fcitx-qt5-gui-wrapper
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/lib/fcitx/qt/libfcitx-quickphrase-editor5.so
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/ca/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/da/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/de/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/es/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/fr/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/ja/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/ko/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/ru/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/zh_CN/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/vzvzv662ll696q6gkiq7n0p3wqm39wj8-fcitx-qt5-1.2.3/share/locale/zh_TW/LC_MESSAGES/fcitx-qt5.mo
-- Installing: /nix/store/wr3hjs0zry8pphrpjwalgfkgbhkjvf61-qtbase-5.12.3-bin/lib/qt-5.12/plugins/platforminputcontexts/libfcitxplatforminputcontextplugin.so
CMake Error at platforminputcontext/cmake_install.cmake:55 (file):
  file INSTALL cannot copy file
  "/build/source/build/platforminputcontext/libfcitxplatforminputcontextplugin.so"
  to
  "/nix/store/wr3hjs0zry8pphrpjwalgfkgbhkjvf61-qtbase-5.12.3-bin/lib/qt-5.12/plugins/platforminputcontexts/libfcitxplatforminputcontextplugin.so".
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)


make: *** [Makefile:106: install] Error 1
note: keeping build directory '/tmp/nix-build-fcitx-qt5-1.2.3.drv-1'
builder for '/nix/store/y4szbd4nv02jpn76y04994a8vrayy924-fcitx-qt5-1.2.3.drv' failed with exit code 2
cannot build derivation '/nix/store/sy3swc3ir0slxnn1nk7nixk8bimgyxfh-fcitx-with-plugins-4.2.9.6.drv': 1 dependencies couldn't be built
note: keeping build directory '/tmp/nix-build-neovim-unwrapped-0.3.8.drv-1'
building '/nix/store/pwl8adj7s1b7jysgkasgzhrf47jzqcdp-hwdb.bin.drv'...
building '/nix/store/hgjsf6m833jvwfjadxqd0w6914yqxfvs-nix.conf.drv'...
cannot build derivation '/nix/store/sy7ic06ahid4bzh6qfiz485yzf71jyqa-system-path.drv': 1 dependencies couldn't be built
note: keeping build directory '/tmp/nix-build-nix.conf.drv-1'
note: keeping build directory '/tmp/nix-build-awesome-4.3.drv-1'
note: keeping build directory '/tmp/nix-build-hwdb.bin.drv-0'
note: keeping build directory '/tmp/nix-build-linux-5.1.0-mptcp_v0.96.0.drv-1'
cannot build derivation '/nix/store/rpn24sfm46wakxanbgy5d4fz6il88npr-xsession-wrapper.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/hq2530sx0pf27l9iqlwr0k867i89sac7-nixos-system-jedha-19.09.git.4225da7.drv': 1 dependencies couldn't be built
error: build of '/nix/store/hq2530sx0pf27l9iqlwr0k867i89sac7-nixos-system-jedha-19.09.git.4225da7.drv' failed
sudo nixos-rebuild -I  -I nixpkgs=/home/teto/nixpkgs -I  switch  11,44s user 0,90s system 17% cpu 1:10,56 total
```
**To Reproduce**
`nix build libsForQt5.fcitx-qt5  -f ~/nixpkgs `

**Metadata**
nixos unstable

--- a/pkgs/tools/inputmethods/fcitx-engines/fcitx-libpinyin/default.nix
+++ b/pkgs/tools/inputmethods/fcitx-engines/fcitx-libpinyin/default.nix
@@ -25,8 +25,6 @@ stdenv.mkDerivation rec {
   preInstall = ''
     substituteInPlace src/cmake_install.cmake \
       --replace ${fcitx} $out
-    substituteInPlace po/cmake_install.cmake \
-      --replace ${fcitx} $out
     substituteInPlace data/cmake_install.cmake \
       --replace ${fcitx} $out
     substituteInPlace dictmanager/cmake_install.cmake \

--- a/pkgs/tools/inputmethods/fcitx/fcitx-qt5.nix
+++ b/pkgs/tools/inputmethods/fcitx/fcitx-qt5.nix
@@ -1,6 +1,12 @@
-{ stdenv, fetchFromGitLab, cmake, fcitx, pkgconfig, qtbase, extra-cmake-modules }:
+{ lib, mkDerivation, fetchFromGitLab
+, cmake
+, extra-cmake-modules
+, fcitx
+, pkgconfig
+, qtbase
+}:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
   pname = "fcitx-qt5";
   version = "1.2.3";
 
@@ -17,12 +23,12 @@ stdenv.mkDerivation rec {
 
   preInstall = ''
     substituteInPlace platforminputcontext/cmake_install.cmake \
-      --replace ${qtbase.out} $out
+      --replace ${qtbase.bin} $out
     substituteInPlace quickphrase-editor/cmake_install.cmake \
       --replace ${fcitx} $out
   '';
 
-  meta = with stdenv.lib; {
+  meta = with lib; {
     homepage    = https://gitlab.com/fcitx/fcitx-qt5;
     description = "Qt5 IM Module for Fcitx";
     license     = licenses.gpl2;

